src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="form__field">
+        <legend>Personal information</legend>
+        <label class="form__label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form__label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form__label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form__label">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="form__label last">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="acceptTerm"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__field">
+        <legend>Registration</legend>
+        <label class="form__label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form__label last">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="12"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__field">
+        <legend>An interesting fact about you!</legend>
+        <label class="form__label">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="Yes"
+            >Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="No"
+            >No
+          </label>
```

Comment:
Because it's entirely wrapped in label when user clicks anywhere in this row (except on "No") it selects "Yes".